crowddata/mixins/adpem/view/show_stats.py

```diff
@@ -39,28 +41,26 @@ def get_count(arr: List["MonsterModel"], *elements: "MonsterModel") -> str:
 
     @staticmethod
     def embed(state: ClosableEmbedViewState, props: ShowStatsViewProps):
-
         fields = []
         for mon, c in Counter(props.valid).most_common(6 if len(props.valid) == 6 else 5):
             fields.append(EmbedField(mon.name_en, Box(
                 LabeledText("Net", ShowStatsView.get_count(props.total, mon)),
                 LabeledText("Adj", ShowStatsView.get_count(props.adj, mon)),
                 LabeledText("You", ShowStatsView.get_count(props.you, mon))
             ), inline=True))
-        if len(set(props.valid)) > 6:
+        if len(props.valid) > 6:
```

Comment:
i realize this wasn't done in this pr but we should make 6 a class constant called MAX_EXPANDABLE_RESULTS